Remotion/Web/UnitTests/Core/UI/Controls/ListMenuImplementation/Rendering/ListMenuRendererTest.cs

```diff
@@ -87,7 +87,7 @@ public void Render_RegistersMenuItems ()
     {
       SetUpGetPostBackLinkExpectations(false);
 
-      string script = "ListMenu.Initialize ('#{0}');\r\n" +
+      string script = $"ListMenu.Initialize ('#{{0}}');{Environment.NewLine}" +
                       "ListMenu.AddMenuInfo ('#{0}', \r\n\tnew ListMenu_MenuInfo ('{0}', new Array (\r\n" +
                       "\t\t{1},\r\n\t\t{2},\r\n\t\t{3},\r\n\t\t{4} ) ) );\r\n" +
```

Comment:
since there's more newline char, I would suggest reverting and fixing the one spot in the prod-code to use hardcoded newlines rather than mixing it here in the test with chaotic readability